deploy/olm-catalog/smart-gateway-operator/manifests/smart-gateway-operator.clusterserviceversion.yaml

```diff
@@ -185,14 +185,6 @@ spec:
           - update
           - patch
           - watch
-        - apiGroups:
-          - ""
-          resources:
-          - namespaces
```

Comment:
This was a result of our previous attempt at alignment with openshift-monitoring?

Reply:
Yes.